graphql/ui_plugin_build_baron.go

```diff
@@ -115,21 +115,35 @@ func BbGetCreatedTicketsPointers(taskId string) ([]*thirdparty.JiraTicket, error
 	return results, nil
 }
 
-func GetSearchReturnInfo(taskId string, exec string) (*thirdparty.SearchReturnInfo, bool, error) {
-	projectNotFoundError := false
+type buildBaronConfig struct {
+	ProjectFound     bool
+	SearchConfigured bool
+}
+
+func GetSearchReturnInfo(taskId string, exec string) (*thirdparty.SearchReturnInfo, buildBaronConfig, error) {
+	bbConfig := &buildBaronConfig{}
 	t, err := BbGetTask(taskId, exec)
 	if err != nil {
-		return nil, projectNotFoundError, err
+		return nil, *bbConfig, err
 	}
 	settings := evergreen.GetEnvironment().Settings()
 	buildBaronProjects := BbGetConfig(settings)
 	bbProj, ok := buildBaronProjects[t.Project]
 
 	if !ok {
-		projectNotFoundError = true
-		return nil, projectNotFoundError, errors.New(fmt.Sprintf("Build Baron project for %s not found", t.Project))
+		bbConfig.ProjectFound = false
+		return nil, *bbConfig, errors.New(fmt.Sprintf("Build Baron project for %s not found", t.Project))
```

Comment:
use errors.Errorf to format new errors, rather than errors.New(fmt.Sprintf()) (sorry this is old code I'm correcting), same with the rest of the function